src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -340,6 +340,16 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         }
     }
 
+    private String convertFromHtml(final String textToConvert) {
```

Comment:
Added the check. It looks for non-ascii in both the preamble and in the argument definitions (not in names though). If non-ascii is found, throws an AssertionError (seemed appropriate, happy to change the type if someone has a better idea) that reads either e.g.

    java.lang.AssertionError: Non-ASCII character used in documentation of CollectInsertSizeMetrics (DEVIATIONS). Only ASCII characters are allowed.

for an argument definition or

    java.lang.AssertionError: Non-ASCII character used in documentation of CollectInsertSizeMetrics (preamble). Only ASCII characters are allowed.

for a tool doc preamble found in violation of the rule.

To test, add ascii characters into your favorite tool doc and try to build.